specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-01-23-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,2774 @@
+        },
+        "applicationGroupReferences": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of applicationGroup links.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "ssoContext": {
+          "type": "string",
+          "description": "Path to keyvault containing ssoContext secret."
+        }
+      }
+    },
+    "HostPoolPatch": {
+      "type": "object",
+      "description": "HostPool properties that can be patched.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "tags to be updated"
+        },
+        "properties": {
+          "description": "HostPool properties that can be patched.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/HostPoolPatchProperties"
+        }
+      }
+    },
+    "HostPoolPatchProperties": {
+      "description": "Properties of HostPool.",
+      "type": "object",
+      "properties": {
+        "friendlyName": {
+          "type": "string",
+          "description": "Friendly name of HostPool."
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of HostPool."
+        },
+        "customRdpProperty": {
+          "type": "string",
+          "description": "Custom rdp property of HostPool."
+        },
+        "maxSessionLimit": {
+          "type": "integer",
+          "description": "The max session limit of HostPool."
+        },
+        "loadBalancerType": {
+          "enum": [
+            "BreadthFirst",
+            "DepthFirst",
+            "Persistent"
+          ],
+          "x-ms-enum": {
+            "name": "LoadBalancerType",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "The type of the load balancer."
+        },
+        "ring": {
+          "type": "integer",
+          "description": "The ring number of HostPool."
+        },
+        "validationEnvironment": {
+          "type": "boolean",
+          "description": "Is validation environment."
+        },
+        "registrationInfo": {
+          "$ref": "#/definitions/RegistrationInfoPatch",
+          "description": "The registration info of HostPool."
+        },
+        "ssoContext": {
+          "type": "string",
+          "description": "Path to keyvault containing ssoContext secret."
+        }
+      }
+    },
+    "RegistrationInfo": {
+      "description": "Represents a RegistrationInfo definition.",
+      "type": "object",
+      "properties": {
+        "expirationTime": {
+          "readOnly": true,
+          "format": "date-time",
+          "type": "string",
+          "description": "Expiration time of registration token."
+        },
+        "token": {
+          "readOnly": true,
+          "type": "string",

```

Comment:
Is this meant to be consumed as a string, or as a byte array?  If string this is fine, otherwise, you can add format byte, which indicated this is a base64 encoded string that should be treated as a byte array.

Reply:
ok it is